script/bulk_import/base.rb

```diff
@@ -803,6 +803,15 @@ def discourse_reaction_id_from_original_id(id)
 
   GAMIFICATION_SCORE_EVENT_COLUMNS = %i[user_id date points description created_at updated_at]
 
+  SOLVED_SOLVED_TOPICS_COLUMNS = %i[
+    topic_id
+    answer_post_id
+    accepter_user_id
+    topic_timer_id
```

Comment:
```suggestion
```
`TopicTimer` isn’t supported in the current iteration of the script, this won’t be imported, safe to omit